src/zoteroOverlay.jsx

```diff
@@ -153,6 +180,8 @@ const zoteroOverlay = {
         this.switcherObserver.disconnect();
 
         this.uninstallTranslators();
+
+        delete Zotero.Server.Endpoints["/zotero-cita/test"];
```

Comment:
probably you mean `delete Zotero.Server.Endpoints["/zotero-cita/add-citations"] `?

Reply:
Good catch, thanks!